custom/conf/app.example.ini

```
@@ -1025,6 +1025,9 @@ LEVEL = Info
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; List of reasons why a Pull Request or Issue can be locked
 ;LOCK_REASONS = Too heated,Off-topic,Resolved,Spam
+;; Maximum number of pinned Issues
+;; Set to 0 to disable pinning Issues
+;MAX_PINNED = 6
```

```suggestion
;MAX_PINNED = 3
```

Also add an entry to https://github.com/go-gitea/gitea/blob/main/docs/content/doc/administration/config-cheat-sheet.en-us.md please. Every config addition should be both in `app.example.ini` and there.